pages/tool_assembly/xnat_pic_tool_assembly.md

```diff
@@ -0,0 +1,52 @@
+---
+title: XNAT-PIC - Extending XNAT to Preclinical Imaging Centers
+search_exclude: true
+contributors: [Sara Zullino, Alessandro Paglialonga, Walter Dastrù, Dario Longo, Silvio Aime.]
+tags: [Preclinical  Imaging,  XNAT, Magnetic  Resonance  Imaging,  Image  Processing,  OpenScience, Database]
+page_tag: <!---REPLACE THIS with the tag that is used in the tool table--->
+---
+
+## What is XNAT-PIC?
+
+**XNAT for Preclinical Imaging Centers (XNAT-PIC)** consists of a set of tools to store, process and share preclinical imaging studies built on top of the [XNAT](https://www.xnat.org/) imaging informatics platform.
+
+
+ 
+## Who is XNAT-PIC intended for?
+
+Preclinical imaging centers deal with many challenges mostly related to the variety of imaging instrumentation yielding huge volumes of raw data. The current procedures to collect, share and reuse preclinical image data are insufficient, thus revealing an urgent need of standardization in terms of data storage and image processing. XNAT-PIC has been developed to overcome this limitation by extending XNAT’s basic functionalities to meet the needs of preclinical imaging facilities. 
+
+## Which task can be solved with XNAT-PIC?
```

Comment:
Maybe this could be a nice addition?
![image](https://user-images.githubusercontent.com/58505673/125260095-44e07d00-e300-11eb-9480-84bd4b970c58.png)
Alternatively you could fit the tool assembly into this template: https://github.com/elixir-europe/rdmkit/blob/master/images/optional_template_infrastructure_tool_assembly.svg

Reply:
Yes, definitely. I have added a sligthly different figure.